### PR TITLE
Add protocol V5.

### PR DESCRIPTION
**Pull request by exazg**

### CODE REVIEW

File: docs/WebsocketAPIV5.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+## Subconnections
+
+### Introduction
+
+Subconnections are additional connections to Exasol cluster nodes which can be created by the client. There are two main reasons to create and use subconnections.
+1. Read a result set in parallel: [fetch](commands/fetchV1.md) can be called in parallel by each of the subconnections to read a result set.
+2. `INSERT` data in parallel: [executePreparedStatement](commands/executePreparedStatementV1.md) can be called in parallel on each of the subconnections to `INSERT` various data. Please note that the same prepared statement (see [createPreparedStatement](commands/createPreparedStatementV1.md)) must be executed on all subconnections.
+
+ℹ️ Here, parallel refers to the node-wise reading/inserting of data. For example, if there is a subconnection for each node (i.e., the number of subconnections equals the number of cluster nodes), then each subconnection will read/insert data locally from/into its node.
```
Comment (exa-mohamedi) [inline docs/WebsocketAPIV5.md:195]: I think, I needs to be removed in the beginning of the statement. And also in previous versions too.

File: docs/WebsocketAPIV5.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+WebSocket Protocol v5 requires an Exasol version of at least 8.32.0. It follows the RFC 6455 document of the IETF.
+
+The Exasol connection server identifies the initial GET request by the client.
+This request contains information about the used protocol version.
+Depending on this information the matching login and protocol class is
+chosen.
+
+After the handshake the process is identical to a connection using the
+standard drivers like JDBC or ODBC: The connection server listens to
```
Comment (exa-mohamedi) [inline docs/WebsocketAPIV5.md:11]: `JDBC or ODBC: The connection`

Colon is intended or Period

File: docs/WebsocketAPIV5.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+* [Data Types: Type names and properties](#data-types-type-names-and-properties)
+* [Compression](#compression)
+* [Heartbeat/Feedback messages](#heartbeatfeedback-messages)
+* [Subconnections](#subconnections)
+
+## Changes
+
+| Date | Exasol Version | Change | Issue |
+| --- | --- | --- | --- |
+| 2024.09.05 | 8.32.0 | The latest update to Exasol enables the use of TIMESTAMP data types with various precisions. | |
```
Comment (exa-mohamedi) [inline docs/WebsocketAPIV5.md:27]: May I know, What does this date refers to?

File: docs/WebsocketAPIV5.md [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+| BOOLEAN | | |
+| CHAR | size | |
+| DATE | | |
+| DECIMAL | precision, scale | |
+| DOUBLE | | |
+| GEOMETRY | | |
+| HASHTYPE | | |
+| INTERVAL DAY TO SECOND | precision, fraction | |
+| INTERVAL YEAR TO MONTH | precision | |
+| TIMESTAMP | | precision, withLocalTimeZone |
```
Comment (exa-mohamedi) [inline docs/WebsocketAPIV5.md:142]: Does the default precision needs to be specified?

Comment (exazg) [reply docs/WebsocketAPIV5.md:142]: No, we don't specify any values--just the API/format.

File: docs/WebsocketAPIV5.md (hunk 3 above)
Comment (exazg) [reply docs/WebsocketAPIV5.md:27]: Date up publishing/release. They'll have to be adjusted.

File: docs/WebsocketAPIV5.md (hunk 2 above)
Comment (exazg) [reply docs/WebsocketAPIV5.md:11]: Colon. ;-)

File: docs/WebsocketAPIV5.md (hunk 1 above)
Comment (exazg) [reply docs/WebsocketAPIV5.md:195]: Yeah, they used to be info emojis, but they were somehow removed.

**Pull request merged**